_includes/shared/who-we-are.html

```diff
@@ -4,9 +4,13 @@
         <h4>Who we are</h4>
 
         <p>
-            We are Adobe Consulting Services.
-            We build solutions on top of Adobe products including Adobe Experience Manager (AEM, fka CQ, fka Communique).
-            <a href="http://www.adobe.com/consulting/" target="_blank">Learn more.</a>
+            We are the AEM developer community.
```

Comment:
Feels presumptuous. IMO the next line is sufficient.

Reply:
sure